management-app/pom.xml

```diff
@@ -50,6 +50,12 @@
             <artifactId>tomcat-embed-jasper</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.nearsoft.fgaribay</groupId>
+            <artifactId>common</artifactId>
+            <version>1.0-SNAPSHOT</version>
```

Comment:
Remove `<version>0.0.1-SNAPSHOT</version>` at the start of this file and you can specify the version for this dependency like this:

```xml
<dependency>
    <groupId>com.nearsoft.fgaribay</groupId>
    <artifactId>common</artifactId>
    <version>${project.version}</version>
</dependency>
```